pages/linux/ncdu.md

```diff
@@ -0,0 +1,19 @@
+# ncdu
+
+> Disk usage analyzer with an ncurses interface.
+
+- Analyze current working directory:
+
+`ncdu`
+
+- Analyze given directory:
+
+`ncdu directory/`
+
+- Save results to file:
+
+`ncdu -o file`
```

Comment:
Could `file` be `{{path/to/file}}` please?

Reply:
done